nodl_python/nodl/errors.py

```diff
@@ -67,6 +73,8 @@ def __init__(self, attribute: str, element: etree._Element) -> None:
         super().__init__(
             f'Value: {element.get(attribute)} is not valid for attribute {attribute}', element
         )
+        self.attribute = attribute
+        self.value = element.get(attribute)
```

Comment:
Ditto above, just save the element and attribute and let the caller sort it out.

Reply:
97698c7